### DISCUSSION TITLE
Translate MDN content to Hindi?

### DISCUSSION THREAD

**Discussion by himanshugarg** [mdn/mdn-community#599] [Content] [answered]
Hi,

From here - https://hacks.mozilla.org/2021/02/mdn-localization-update-february-2021/

>  After we have unfrozen the “tier 1” locales (fr, ja, zh-CN, zh-TW), we are planning to stop displaying other locales

If we found a localization team, would [Hindi](https://www.infoplease.com/world/social-statistics/most-widely-spoken-languages-world) be a language of interest? If yes, are there tools that the MDN translation community uses to get an initial machine translated version that preserves the macros and markup? 

Thanks

**Comment by himanshugarg** [comment, marked as answer]
Already answered here - https://github.com/mdn/content/discussions/5627

**Comment by pepelsbey** [comment]
Closing as answered/settled.
